keymap_drawer/parse/rmk.py

```diff
@@ -0,0 +1,408 @@
+"""Parsing for RMK TOML keymap files."""
+
+import logging
+import re
+try:
+    import tomllib
+except ImportError:
+    # For Python < 3.11, would need to install tomli
+    # But since this project requires Python 3.12+, this should not happen
+    raise ImportError("tomllib not available - RMK parsing requires Python 3.11+")
+
+from pathlib import Path
+from typing import Sequence
+
+from keymap_drawer.keymap import ComboSpec, KeymapData, LayoutKey
+
+from .parse import KeymapParser, ParseError
+
+logger = logging.getLogger(__name__)
+
+
+class RmkKeymapParser(KeymapParser):
+    """Parser for RMK TOML keymap files."""
+
+    _modifier_fn_to_std = {
+        "LCtrl": ["left_ctrl"],
+        "LShift": ["left_shift"],
+        "LAlt": ["left_alt"],
+        "LGui": ["left_gui"],
+        "RCtrl": ["right_ctrl"],
+        "RShift": ["right_shift"],
+        "RAlt": ["right_alt"],
+        "RGui": ["right_gui"],
+    }
+
+    def __init__(
+        self,
+        config,
+        columns: int | None,
+        base_keymap: KeymapData | None = None,
+        layer_names: list[str] | None = None,
+        virtual_layers: list[str] | None = None,
+    ):
+        super().__init__(config, columns, base_keymap, layer_names, virtual_layers)
+        self.hold_taps = {"MT": ["&kp", "&kp"], "LT": ["&mo", "&kp"]}
+
+    def _str_to_key(
+        self, binding: str, current_layer: int | None, key_positions: Sequence[int], no_shifted: bool = False
+    ) -> LayoutKey:
+        """Convert an RMK key binding string to a LayoutKey."""
+        if binding in self.raw_binding_map:
+            return LayoutKey.from_key_spec(self.raw_binding_map[binding])
+        
+        if self.cfg.skip_binding_parsing:
+            return LayoutKey(tap=binding)
+
+        assert self.layer_names is not None
+        assert self.layer_legends is not None
+
+        def mapped(key: str) -> LayoutKey:
+            """Map a keycode to a LayoutKey using the configuration."""
+            if entry := self.cfg.zmk_keycode_map.get(key):
+                return LayoutKey.from_key_spec(entry)
+            key, mods = self.parse_modifier_fns(key)
+            
+            # Map common RMK keycodes to standard ones
+            rmk_keycode_map = {
+                "Kp0": "KP_0", "Kp1": "KP_1", "Kp2": "KP_2", "Kp3": "KP_3", "Kp4": "KP_4",
+                "Kp5": "KP_5", "Kp6": "KP_6", "Kp7": "KP_7", "Kp8": "KP_8", "Kp9": "KP_9",
+                "KpEnter": "KP_ENTER", "KpDot": "KP_DOT",
+                "KbVolumeDown": "VOL_DN", "KbVolumeUp": "VOL_UP", "KbMute": "MUTE",
+                "Dot": ".", "Comma": ",", "Quote": "'", "Backspace": "BSPC",
+                "Enter": "RET", "Space": "SPC", "__": "",
+            }
+            
+            display_key = rmk_keycode_map.get(key, key)
+            mapped = LayoutKey.from_key_spec(
+                self.cfg.zmk_keycode_map.get(display_key, display_key)
```

Comment:
Does `zmk_keycode_map` work for RMK keycodes in general? I assume not, I think it's a bit hacky and difficult to explain to users. Can you create a short `rmk_keycode_map` with above contents as default in config.py?